todo_app/trello_items.py

```diff
@@ -0,0 +1,42 @@
+from flask import trello
+import requests
+import json
+
+
+class Item:
+    def __init__(self, id, name, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], list['name'])
+
+
+def add_items():
+    
+    return trello.get('items')
+
+def get_item(id):
+
+    item = Item.from_trello_card(list)
+
+url = "https://api.trello.com/1/boards/b3L47Acw/lists"
+
+headers = {
+  "Accept": "application/json"
+}
+
+query = {
+  'key': 'APIKey',
+  'token': 'APIToken'
```

Comment:
This is the right structure, and possibly you've just removed your secrets before committing so that they don't end up in source control, but we really want these values to be the ones that actually give us access to the API without committing them to Git, so we want something like:
```suggestion
  'key': os.getenv('TRELLO_KEY'),
  'token': os.getenv('TRELLO_TOKEN')
```

This way we can set the appropriate variables in our `.env` file